client/src/components/Button/ThankButton.js

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import styled from "styled-components";
+import { mq, theme } from "constants/theme";
+
+const primaryColor = theme?.colors?.royalBlue;
+const whiteColor = theme?.colors?.offWhite;
+
+const ThanksButton= styled.div`
+display: flex;
+flex-direction: row;
+justify-content: center;
+align-items: center;
+padding: 8px 38px;
```

Comment:
@linadinh2602 just pointed out on my code we should be using rems instead of px, I agree :) 

I guess the question would be what if the figma designs have px in them...maybe it's ok then??

Reply:
ok!